zerver/webhooks/netlify/view.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, Dict, Iterable, Optional
+
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.webhooks.common import check_send_webhook_message, \
+    validate_extract_webhook_http_header, UnexpectedWebhookEventType
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_error, json_success
+from zerver.lib.validator import check_dict, check_string
+from zerver.models import UserProfile
+
+EVENTS = ['deploy_failed', 'deploy_locked', 'deploy_unlocked', 'deploy_building', 'deploy_created']
+
+@api_key_only_webhook_view('Netlify')
+@has_request_variables
+def api_netlify_webhook(
+        request: HttpRequest, user_profile: UserProfile,
+        payload: Dict[str, Iterable[Dict[str, Any]]]=REQ(argument_type='body')
+) -> HttpResponse:
+
+    message_template = get_template(request, payload)
+
+    body = message_template.format(build_name=payload['name'],
+                                   build_url=payload['url'],
+                                   branch_name=payload['branch'],
+                                   state=payload['state'])
+
+    topic = "{topic}".format(topic=payload['name'])
```

Comment:
Okay, I just thought of this. I think the topic should be the branch name, not the build name. It makes much more sense to group build notifications for the same branch under the same topic! :)

Also, make sure you update the message screenshot after making this change! :)